### ISSUE TITLE
Incorrect theme path?

### ISSUE THREAD

**Issue by jasteele12** [concrete5/addon_migration_tool_legacy#1]
Hi Andrew,

I just tried this on one of my development servers but received this error (path shortened):

An unexpected error occurred.
File /.../packages/theme_slate/themes/**default**/default.php not found. All themes need default.php and view.php files in them. Consult concrete5 documentation on how to create these files.

That **default** above should be **slate** in this case.

Dev site is 5.6.4.3,

John

**Comment by aembler** [comment]
Yeah, this is valid. I believe the code is in the rendering function that we use in 5.6.3.4 to render a page prior to exporting it. Something about that code attempts to render the page from within the dashboard theme, causing problems. This will need to get fixed.
